src/components/Footer/styles.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+
+export const Row = styled.div`
+  min-width: 300px;
+`;
+
+export const Link = styled.a`
+  & > svg {
+    fill: ${(props) => props.theme.hue.gold};
+  }
+`;
+
+export const TextLink = styled.a`
+  color: ${(props) => props.theme.hue.gold};
+  font-size: 20px;
+  margin: 10px;
+  text-decoration: none;
+`;
+
+export const Icon = styled.img`
+  float: left;
+  display: block;
+`;
+
+export const Logo = styled.img`
+  display: block;
+  max-height: 86px;
+  max-width: 86px;
+  text-transform: capitalize;
+`;
+
+export const Center = styled.p`
+  text-align: center;
+  justify-content: center;
```

Comment:
You shouldn't need `justify-content: center` to align the text to the center. Plus, that property only works if you have `display: flex`.